targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L476xG/TARGET_DISCO_L476VG/PinNames.h

```diff
@@ -142,6 +142,8 @@ typedef enum {
     JOYSTICK_UP = PA_3,
     JOYSTICK_DOWN = PA_5,
     USER_BUTTON = JOYSTICK_CENTER,
+    // Standardized button names
+    BUTTON1 = USER_BUTTON,
```

Comment:
Huh, had never considered how to handle joysticks! Could potentially even add `BUTTON2-5` for each direction, though I think ST could always add that if they wanted to. I think you've handled it nicely here.